test/UnbrandedProjects/NoTest-TypeSpec/src/Generated/Internal/ClientUriBuilder.cs

```diff
@@ -0,0 +1,204 @@
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NoTestTypeSpec
+{
+    internal class ClientUriBuilder
```

Comment:
Just wondering because of my lack of familiarity with autorest.csharp code -- have we added tests to validate all the different scenarios that this type addresses?  If so, that's great - would you be able to share a link for my learning?  This is a non-blocking comment, but anything you provide will help me understand whether we have good validation of the helper type implementation.

Reply:
No, we have very little testing in the unbranded path but the tests should be added here like we did for TypeFormatter https://github.com/Azure/autorest.csharp/blob/feature/v3/test/AutoRest.Shared.Tests/TypeFormatterTests.cs